PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/GraphInterface.java

```diff
@@ -10,5 +14,5 @@
 
   AbstractNode getRoot();
 
-  // public HashMap<Integer, AbstractNode> getNodes();
+  HashMap<Integer, AbstractNode> getAbstractNodes();
```

Comment:
Add comments to the methods in this interface? (then the comments will be automatically inherited by classes which implement this interface, so you will only have to write them for the parent class, unless if you want to add extra details for a specific implementation of the interface of course)